searchcore/src/vespa/searchcore/proton/test/disk_mem_usage_notifier.h

```diff
@@ -19,7 +19,8 @@ class DiskMemUsageNotifier : public IDiskMemUsageNotifier
     std::vector<IDiskMemUsageListener *> _listeners;
     DiskMemUsageState _state;
 public:
-    DiskMemUsageNotifier() : IDiskMemUsageNotifier(), _listeners(), _state() { }
+    DiskMemUsageNotifier() : IDiskMemUsageNotifier(), _listeners(),
+                             _state({ 0.8, 0.5 }, { 0.8, 0.4 }) { }
```

Comment:
Perhaps it is better to take these disk and memory parameters as constructor arguments in a new contructor? It was hard to understand why ResourceUsageTrackerTest::resource_usage_is_forwarded_to_listener() got the values 0.5 and 0.4 after just adding a listener to the tracker.